tests/interoperability_test.py

```diff
@@ -417,35 +393,34 @@ def test_run_evaluation_for_cardinality_estimator_workflow_compatible(self):
     the evaluation, analyzes results, and generates a report, which should not
     run into any error.
     """
-    # Run evaluation.
-    evaluator_out_dir = self.create_tempdir('evaluator').full_path
-    evaluation_run_name = 'interoperability_test_for_evaluator'
-    generate_results = evaluator.Evaluator(
-        evaluation_config=self.evaluation_config,
-        sketch_estimator_config_list=self.sketch_estimator_config_list,
-        run_name=evaluation_run_name,
-        out_dir=evaluator_out_dir,
-        workers=0)
-    generate_results()
-
-    # Run analysis.
-    analysis_out_dir = self.create_tempdir('analyzer').full_path
-    generate_summary = analyzer.CardinalityEstimatorEvaluationAnalyzer(
-        out_dir=analysis_out_dir,
-        evaluation_directory=evaluator_out_dir,
-        evaluation_run_name=evaluation_run_name,
-        evaluation_name=self.evaluation_config.name,
-        estimable_criteria_list=[(0.1, 0.9)])
-    generate_summary()
-
-    # Generator report.
-    report_out_dir = self.create_tempdir('report').full_path
-    generate_report = report_generator.CardinalityReportGenerator(
-        out_dir=report_out_dir,
-        analysis_out_dir=analysis_out_dir,
-        evaluation_run_name=evaluation_run_name,
-        evaluation_name=self.evaluation_config.name)
-    generate_report()
+    # TODO: the bloom filter with the union estimator doesn't work with the
+    # global noiser. Need to fix this.
+    run_evaluation._run(
+        run_evaluation=True,
+        run_analysis=True,
+        generate_html_report=True,
+        evaluation_out_dir=self.create_tempdir('evaluator').full_path,
+        analysis_out_dir=self.create_tempdir('analyzer').full_path,
+        report_out_dir=self.create_tempdir('report').full_path,
+        evaluation_config='smoke_test',
+        sketch_estimator_configs=[
+            'log_bloom_filter-100000-first_moment_log-infty-0.2747',
+            'exp_bloom_filter-250000_10-first_moment_exp-0.1099-0.2747',
+            'vector_of_counts-4096-sequential-1.0986-infty',
+            'exact_set-None-independent_estimator_universe100000-infty-0.2747',
+            # 'bloom_filter-5000000_hash8-union_estimator-infty-0.2747',
```

Comment:
I believe this is because epsilon was not applied on this when pr38 was merged, I can add a PR to fix